tokens/src/mock.rs

```diff
@@ -36,7 +45,7 @@ parameter_types! {
 	pub const AvailableBlockRatio: Perbill = Perbill::one();
 }
 
-type AccountId = u128;
+type AccountId = u64;
```

Comment:
Why changing it to `u64`?

Reply:
```
impl Contains<u64> for TenToFourteen {
```
Other defines are u64, it is more convenient to modify here